README.md

```diff
@@ -1,5 +1,109 @@
-# Codess in the classroom attendee material
+# Codess In The Classroom
 
-Repository to store the specs and instructions to use in the classroom.
+Welcome! This repository contains the Codess In The Classroom course
+materials. From here you will be able to find (and contribute to!) the
+materials we use in our hands-on workshops.
 
-1. [Week 1](./week1-webclient.md)
+## Introduction
+
+The course is going to be structured as a series of workshops, during which
+you will learn about various technologies and practices used in modern
+software development.
+
+In each workshop, you and your peers will be the development team working on
+part of a web application for a pet adoption center. You will learn about
```

Comment:
nit: Should we use the term "animal shelter" here rather than "pet adoption center"? (which sounds a lot more 1984) :smile:

Reply:
😂👍